frontend/amundsen_application/config.py

```diff
@@ -32,8 +32,13 @@ class Config:
 
     COLUMN_STAT_ORDER = None  # type: Dict[str, int]
 
+    # The following three variables control whether table descriptions can be edited via the UI
+    # ALL_UNEDITABLE_SCHEMAS: set environment variable to 'true' if you don't want any schemas to be able to be edited
+    # UNEDITABLE_SCHEMAS: a set of schema names whose tables will not be editable
+    # UNEDITABLE_TABLE_DESCRIPTION_MATCH_RULES: a list of regex rules for schema name, table name, or both
+    # See https://www.amundsen.io/amundsen/frontend/docs/flask_config/#uneditable-table-descriptions for more info!
+    ALL_UNEDITABLE_SCHEMAS = os.getenv('ALL_UNEDITABLE_SCHEMAS', 'false') == 'true'  # type: bool
```

Comment:
nit - we may want to do a little bit of processing on the envvar just to allow for True, true, 1 (or 0) etc. that are common values that are used as booleans

Reply:
Yeah I thought about this but I noticed a similar pattern on [Line 77-78](https://github.com/amundsen-io/amundsen/pull/1353/files/f4c6f039b7bb73f3964af8c76f45e1df2a37eb4e#diff-e6f76b9d9261ca6c32f284a0a78c47c70d36fe887a9dace1de9ae1bb672d8d78R77-R78):
```python
# Settings for Preview Client integration
    PREVIEW_CLIENT_ENABLED = os.getenv('PREVIEW_CLIENT_ENABLED') == 'true'  # type: bool
```

I think a utility function might be nice here but seeing as I documented the exact value that the env var needs to be I think this is ok for now. What do you think?